### PR TITLE
Align jumbotron sizes

### LINKED ISSUE #250: [design QA] hero banners
All banners images should have the same height and text should have the same width and style.

Banners appear on:
- home
- fellowships
- study groups
- projects
- resources

Home, Projects, and Resources banners should be taller. Right now they're about 275px but they should be more like 360px (like on Fellowship page).

Staging:
screen shot 2016-04-25 at 10 33 20 am

Comp:
screen shot 2016-04-25 at 10 40 53 am

All the body text in Project's banner should be the same as on the home page banner.
Staging:
screen shot 2016-04-25 at 10 33 29 am

Comp:
screen shot 2016-04-25 at 10 41 09 am

All text should have the same max width, Resources is too long. 

screen shot 2016-05-25 at 2 07 56 pm

### PR DESCRIPTION
**Pull request by alanmoo**
Closes #250 

And adjusts size of "What's New" on homepage. See below for review app, @kristinashu.

### CODE REVIEW

**Comment by kristinashu** [comment]
Ah so much better, thanks @alanmoo!

File: app/react/pages/fellowships/fellowships.jsx [hunk 1]
```diff
@@ -28,11 +28,17 @@ export default React.createClass({
...
         <div className="jumbotron text-xs-center jumbotron-fluid m-b-0">
-          <div className="container m-b-3">
-            <h2>Fellowships</h2>
-            <p className="lead m-t-1"><em>Are you a champion of open science and open data? Mozilla is seeking researchers eager to advance openness in science and data within their institutions.</em></p>
-            <div className="btn btn-white">Applications open on June 3</div>
-            <a hidden className="btn btn-white" href="https://mozilla-science-lab.forms.fm/mozilla-fellows-for-science-2016" target="_blank">Apply Now</a>
+          <div className="container-dynamic container m-b-3">
+            <div className="row">
+              <h2 className="col-xs-12">Fellowships</h2>
+              <p className="lead m-t-1 col-xs-12 col-md-push-1 col-md-10 col-lg-8 col-lg-push-2"><em>Are you a champion of open science and open data? Mozilla is seeking researchers eager to advance openness in science and data within their institutions.</em></p>
```
Comment (gvn) [inline app/react/pages/fellowships/fellowships.jsx]: urk

Comment (alanmoo) [inline app/react/pages/fellowships/fellowships.jsx]: ¯\\\_(ツ)_/¯  that's bootstrap for you.

Comment (gvn) [inline app/react/pages/fellowships/fellowships.jsx]: LOL, yep.

**Pull request merged**